common/data_refinery_common/message_queue.py

```diff
@@ -43,6 +43,8 @@ def send_job(job_type: Enum, job) -> None:
         nomad_job = ProcessorPipeline.ILLUMINA_TO_PCL.value
     elif job_type is ProcessorPipeline.SMASHER:
         nomad_job = ProcessorPipeline.SMASHER.value
+    elif job_type is ProcessorPipeline.QN_REFERENCE:
+        nomad_job = ProcessorPipeline.QN_REFERENCE.value
```

Comment:
I believe this means that this will use the `QN_REFERENCE` image. Take a look at the agilent `elif` to see how we get it to use the affy image.